nss_cache/sources/ldapsource.py

```diff
@@ -712,6 +715,10 @@ def Transform(self, obj):
     elif 'member' in obj:
       for member_dn in obj['member']:
         member_uid = member_dn.split(',')[0].split('=')[1]
+        if base and member_dn.endswith(base):
```

Comment:
Please add a unit test for this new code in ldapsource_test.py.

Reply:
Tests added!